Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VARIANT=2.6
+ARG VARIANT=2.7
```

Comment:
The ruby version pinned in .ruby-version is 2.6.2 (and the one used and cached by Netlify).
This change makes the versions out of sync.

Reply:
I'll revert this